addons/mail/demo/discuss/public_channel_demo.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record model="mail.guest" id="mail.guest_alex_demo">
+            <field name="name">Alex</field>
+        </record>
+        <record model="discuss.channel" id="mail.channel_public_community_demo">
+            <field name="name">Public Community</field>
+            <field name="description">A space for engaging discussions among employees, partners, and the public.</field>
+            <field name="group_public_id" eval="False"/>
+        </record>
+        <record model="mail.message" id="mail.channel_public_community_message_0_demo">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="mail.channel_public_community_demo"/>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="date" eval="(DateTime.today() - timedelta(minutes=30)).strftime('%Y-%m-%d %H:%M')"/>
+            <field name="body">We're excited to announce our product launch this September! 🎉
+            We'll be hosting a live video conference on this channel to showcase all the details.
+            If you're interested, feel free to share the link with your customers so they can tune
+            in and watch it live!</field>
+        </record>
+        <record model="mail.message" id="mail.channel_public_community_message_1_demo">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="mail.channel_public_community_demo"/>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="author_guest_id" ref="mail.guest_alex_demo"/>
+            <field name="parent_id" ref="mail.channel_public_community_message_0_demo"/>
+            <field name="date" eval="(DateTime.today() - timedelta(minutes=20)).strftime('%Y-%m-%d %H:%M')"/>
+            <field name="body">That sounds amazing! Can't wait to see what's new.
+            Will there be a QA session during the conference?</field>
+        </record>
+        <record model="mail.message.reaction" id="mail.channel_public_community_message_reaction_demo">
```

Comment:
channel_public_community_message_1_reaction_0_demo

Reply:
Also it would be nice to have a reaction from a guest too, on one of the other messages.